docs/npc-swap-tutorial.md

```diff
@@ -0,0 +1,247 @@
+
+In this case, the default Signature Suit is fairly easy to find, as it's the second result in the chunk0 TEMP tree and matches up with the in-game name (`outfit_agent47_signaturesuit_heroa_v0`). If we look at the result in the left box, we can see it begins with `00FF8C6314EA882E.TEMP`, so the value we're looking for is `00FF8C6314EA882E`. Make a note of this TEMP hash.
+
+Next, we will want to locate the TBLU value. In this case, since we still have the results in chunk0 open, we can simply access the entry in the TBLU tree instead. We can see the TBLU is named `0046BB3BE76661CC.TBLU`, so the value we're looking for is `0046BB3BE76661CC`. Make a note of this TBLU hash as well.
+
+# Modifying the Files (Using Community Resources)
+
+Alternatively, we can use HMBM47's [Hitman 3 Outfits Spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vRDiyiqdRebu0Olvvkr20CDhh6ANxu7FOQZ_O-1YHFN9e6kh0WmpbwDYbfgzevSvc3fO4_4Exu1fmQH/pubhtml#), only this time, we'll access the `chunk0 47 Base Suits` section. As before, we'll see each suit's in-game name in the `Name` column, and the suit's filename in the `Outfit` column. However, we'll need the TEMP and TBLU hashes rather than the filename, so we can locate them in the `TEMP` and `TBLU` columns, respectively. First, if we search for `Signature Suit`, we can identify which is the default version and which is the gloved version by looking at either the `Notes` column, or the filename in the `Outfit` column. Then, we can identify the outfit's TEMP by looking at the TEMP column on the left, which is `00FF8C6314EA882E.TEMP`. Therefore, the TEMP hash we need is `00FF8C6314EA882E`. We can then do for the outfit's TBLU by examining the TBLU column, which gives us `0046BB3BE76661CC.TBLU`. Therefore, the TBLU hash we need is `0046BB3BE76661CC`.
+
+In either case, once you have the TEMP and TBLU hashes, the next step is to open the `00873434CB4F9FCD.entity.json` file (which we previously exported) using a code editor. You'll see it starts with: `{"tempHash":"00873434CB4F9FCD","tbluHash": "00945FFF47BBBDE3",` All we need to do is replace the tempHash and tbluHash values in this string with the ones we obtained in the previous step. Therefore, we would change `"tempHash":"00873434CB4F9FCD"` to `"tempHash":"00FF8C6314EA882E"` and `"tbluHash":"00945FFF47BBBDE3"` to `"tbluHash":"0046BB3BE76661CC"`
+
+The end result should be: `{"tempHash":"00FF8C6314EA882E","tbluHash":"00945FFF47BBBDE3"`

```

Comment:
Missing closing brace.

Reply:
Hmm, I was just quoting the contents of the JSON verbatim while solely including the tempHash and tbluHash section, in order to make it easy for new users to follow. I figured if I added a closing brace, someone without a ton of technical knowledge may attempt to copy/paste the entire string verbatim, which would cause issues.

Reply:
I feel like we shouldn't be writing these guides for people who aren't technical enough to understand how basic JSON works... almost feels like giving a chainsaw to a kindergartener.

Reply:
Sure, that's a fair point. But there may be users out there that are skilled/talented in other areas (3D editing, graphic editing, etc.) that may not be familiar with basic JSON. In this tutorial, users are only modifying existing JSON files, rather than creating their own from scratch, and If they mess up their JSON, the mod simply won't work. And when it comes to editing entities using QNE in another tutorial, QNE will do the work for them.

Maybe it would be worth creating another section that goes over basic JSON syntax? The user can also compare their JSON with functioning examples to see what works and what doesn't (or by trial and error of course, haha).